tests/nested/extra-snaps-assertions/task.yaml

```diff
@@ -0,0 +1,73 @@
+summary: create ubuntu-core image and execute the suite in a nested qemu instance
+
+systems: [ubuntu-16.04-64]
+
+prepare: |
+    . $TESTSLIB/prepare.sh
+    prepare_classic
+    prepare_each_classic
+
+    # determine arch related vars
+    case "$NESTED_ARCH" in
+    amd64)
+        QEMU="$(which qemu-system-x86_64)"
+        ;;
+    i386)
+        QEMU="$(which qemu-system-i386)"
+        ;;
+    *)
+        echo "unsupported architecture"
+        exit 1
+        ;;
+    esac
+
+    # create ubuntu-core image
+    mkdir -p /tmp/work-dir
+    snap install --devmode --beta ubuntu-image
+
+    snap download core
+
+    /snap/bin/ubuntu-image --image-size 3G $TESTSLIB/assertions/nested-${NESTED_ARCH}.model --channel $CORE_CHANNEL --output ubuntu-core.img --extra-snaps core_*.snap
+    mv ubuntu-core.img /tmp/work-dir
+
+    # create assertion block device
+    genisoimage -volid cidata -joliet -rock -o assertions.disk $TESTSLIB/assertions/auto-import.assert
```

Comment:
AFAIK yes, if not the nested instance is not accessible at all, we would need to go through the console-conf initialization, this system-user assertion allows us to access as user1:ubuntu, which is used later to configure the test user required by spread.